### PR TITLE
Two new FAQ topics

### PR DESCRIPTION
**Pull request by strump**
Added new FAQ topics **"How to create a route and start navigation"** and **"I can't hear voice instructions"** from main repo.

Also moved `content/faq/text-to-speech-android-tts` to `content/faq/voice/text-to-speech-android-tts` to have voice instructions questions in a single subfolder:

* `content/faq/voice/`
  * `cant-hear-instructions`
  * `text-to-speech-android-tts`

But this changed URL from `/faq/text-to-speech-android-tts/` to `/faq/voice/text-to-speech-android-tts/`

### CODE REVIEW

File: content/faq/voice/cant-hear-instructions/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+extra:
+  order: 20
+---
+
+**Note:** voice instruction are available for car and cycling routes. For now, you can hear voice instructions only while driving and when the screen is on.
+
+If you can't hear voice instructions:
+
+* Please make sure that your device's volume is not muted. You can change the volume level with your device's volume buttons. To do that, you may need to disable "Change with Buttons" option (if enabled) in the device settings → Sounds.
```
[the remark this answers is not in the document]
Comment (strump) [reply content/faq/voice/cant-hear-instructions/index.md:17]: "Change with buttons" system setting is mentioned in Apple documentation: https://support.apple.com/en-gb/guide/iphone/iphb71f9b54d/ios

iPhone-volume-5

File: content/faq/voice/text-to-speech-android-tts/index.es.md
[the remark this answers is not in the document]
Comment (strump) [reply content/faq/voice/text-to-speech-android-tts/index.es.md:1]: I think https://organicmaps.app/faq/text-to-speech-android-tts/ page has instructions only for Android while `/faq/voice/cant-hear-instructions` page is for all devices. Page https://organicmaps.app/faq/text-to-speech-android-tts/ is large and shouldn't be merged with `/faq/voice/cant-hear-instructions`.

File: content/faq/voice/cant-hear-instructions/index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+* When you turn on Bluetooth, it shouldn't reflect voice instructions. However, we didn't test voice guidance with car audio, it may cause the issues. In this case, you can turn Bluetooth off.
+
+* If you are an Android user and the option is disabled (or some of the supported languages are unavailable), please check system TTS settings.
+
+On Android voice instructions are available in 27 languages: English, Arabic, Chinese (Traditional and Simplified), Czech, Danish, Dutch, Finnish, French, German, Greek, Hindi, Hungarian, Indonesian, Italian, Japanese, Korean, Persian, Polish, Portuguese, Romanian, Russian, Spanish, Thai, Turkish, Ukrainian, Vietnamese.
```
[the remark this answers is not in the document]
Comment (strump) [reply content/faq/voice/cant-hear-instructions/index.md:23]: I compared languages with [strings-tts.xml](https://github.com/organicmaps/organicmaps/blob/master/android/app/src/main/res/values/strings-tts.xml). FAQ is missing languages:

* Catalan, Spanish (Mexico), Basque, Croatian, Swahili, Norwegian, Portuguese (Brasil), Slovak, Swedish, Belarusian, Serbian, Marathi

I created a [spreadsheet](https://docs.google.com/spreadsheets/d/1h2j-em9P4mmETkCGiiR7NEJtX9D7RS1nsS8bCp4p_7c) to compare and translate the list. Will update the FAQ

File: content/faq/map/how-to-create-a-route/index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Once your location is determined on the map, choose your destination.
+You can use one of the following ways:
+
+* tap the search button
+* tap the bookmarks button
+* tap (tap and hold for empty areas) any place on the map
+
+Once you have chosen point of destination, press “Route to” button at the bottom. The route will be created and you will see distance and estimated travel time. You can change the route type by pressing the Car, Pedestrian, Subway, Bike or Ruler icon at the top of the screen. To start following the route tap “Start” button. Press the arrow sign in the lower right corner of the screen and tap Stop to finish the route.
+
+You can choose a different starting point (“Route From“ button)to preview a route, but navigation is only available from your current location.
```
[the remark this answers is not in the document]
Comment (strump) [reply content/faq/map/how-to-create-a-route/index.md]: Fixed

File: content/faq/map/how-to-create-a-route/index.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+You can add up to 100 intermediate points to a route. To add an intermediate point: create a route between start and destination → then tap a point on the map and tap “Add stop”.
```
[the remark this answers is not in the document]
Comment (strump) [reply content/faq/map/how-to-create-a-route/index.md]: Fixed

File: content/faq/map/how-to-create-a-route/index.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+You can change the settings of the car route and select road types that you would like to avoid (tolls, unpaved roads, motorways, ferries). Open the app settings → Routing options → Toggle On needed options.
```
[the remark this answers is not in the document]
Comment (strump) [reply content/faq/map/how-to-create-a-route/index.md]: Fixed

File: content/faq/map/how-to-create-a-route/index.ru.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  order: 40
+---
+
+Убедитесь, что ваше местоположение определено и выберите пункт назначения. Вы можете:
+
+* использовать поиск
+* выбрать место из меток (нажмите на значок звездочки на панели меню)
+* просто нажать (нажать и удерживать для пустой области) любое место на карте
+
+После того как вы выбрали пункт назначения, нажмите кнопку "Сюда" на странице места. Маршрут будет построен и вы увидите расстояние и приблизительное время в пути. Тип маршрута можно изменить, нажав на иконку автомобиля, пешехода, метро, велосипеда или линейки вверху экрана. Для движения по маршруту, нажмите кнопку "Начать". Чтобы закончить движение по маршруту нажмите на значок стрелки в правом нижнем углу экрана и нажмите "Стоп".
```
[the remark this answers is not in the document]
Comment (strump) [reply content/faq/map/how-to-create-a-route/index.ru.md]: Fixed

File: content/faq/voice/cant-hear-instructions/index.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  order: 20
+---
+
+**Note:** voice instruction are available for car and cycling routes. For now, you can hear voice instructions only while driving and when the screen is on.
+
+If you can't hear voice instructions:
+
+* Please make sure that your device's volume is not muted. You can change the volume level with your device's volume buttons. To do that, you may need to disable "Change with Buttons" option (if enabled) in the device settings → Sounds.
+
+* When you turn on Bluetooth, it shouldn't reflect voice instructions. However, we didn't test voice guidance with car audio, it may cause the issues. In this case, you can turn Bluetooth off.
```
[the remark this answers is not in the document]
Comment (strump) [reply content/faq/voice/cant-hear-instructions/index.md]: Added links from '/faq/voice/cant-hear-instructions' to `/faq/voice/text-to-speech-android-tts` for English, Russian and Ukrainian

File: content/faq/voice/cant-hear-instructions/index.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+* If you are an Android user and the option is disabled (or some of the supported languages are unavailable), please check system TTS settings.
```
[the remark this answers is not in the document]
Comment (strump) [reply content/faq/voice/cant-hear-instructions/index.md]: Added links from '/faq/voice/cant-hear-instructions' to `/faq/voice/text-to-speech-android-tts` for English, Russian and Ukrainian

File: content/faq/voice/cant-hear-instructions/index.ru.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+extra:
+  order: 20
+---
+
+
+**Обратите внимание:** голосовые подсказки доступны для автомобильных и велосипедных маршрутов. Сейчас подсказки озвучиваются только во время движения и только при включенном экране.
+
+Если подсказки не слышно:
+
+* Убедитесь, что на устройстве включен звук. Изменить громкость можно с помощью кнопок громкости на устройстве. Чтобы это сделать, может потребоваться отключить опцию "Изменение кнопками" (если она включена) в настройках устройства → Звуки, тактильные сигналы.
```
[the remark this answers is not in the document]
Comment (strump) [reply content/faq/voice/cant-hear-instructions/index.ru.md:18]: Добавил языки из [strings-tts.xml](https://github.com/organicmaps/organicmaps/blob/master/android/app/src/main/res/values/strings-tts.xml)

File: content/faq/voice/cant-hear-instructions/index.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (biodranik) [reply content/faq/voice/cant-hear-instructions/index.md]:
```
11:28:29.034 | Error: Failed to render content of /opt/buildhome/repo/content/faq/voice/cant-hear-instructions/index.ru.md
11:28:29.034 | Error: Reason: Broken relative link `@/ru/faq/voice/text-to-speech-android-tts/index.md` in faq/voice/cant-hear-instructions/index.ru.md
```

[the remark this answers is not in the document]
Comment (strump) [reply content/faq/voice/cant-hear-instructions/index.md]: Sorry. Wrong link. Fixed

File: content/faq/voice/cant-hear-instructions/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (biodranik) [reply content/faq/voice/cant-hear-instructions/index.md]: Google TTS doesn't support Belarusian, for example. So the phrase "Google TTS supports all listed languages except of Arabic and Persian (Farsi)." is incorrect. Does it make sense to replace this paragraph with a link to (localized) https://organicmaps.app/faq/text-to-speech-android-tts/#engines and auto-translated text like "Please check which languages are supported by your configured TTS (voice) engine, some of them are listed in [this table](https://organicmaps.app/faq/text-to-speech-android-tts/#engines).

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**